financial_transcripts/datasets/evalset.csv

```diff
@@ -1,4 +1,4 @@
-chat_history,question,answer
```

Comment:
Would you please use question instead of query to be consistent with other flows. This breaks all the evaluation notebook.